src/ogdf/fileformats/DotParser.cpp

```diff
@@ -639,12 +639,12 @@ Ast::Graph *Ast::parseGraph(
 	}
 
 	if(curr == m_tend || curr->type != Token::Type::leftBrace) {
-		// std::cerr << "ERROR: Expected \""
-		//      << Token::toString(Token::leftBrace)
-		//      << ", found \""
-		//      << Token::toString(a->type)
-		//      << "\" at "
-		//      << a->row << ", " << a->column << ".\n";
+        // GraphIO::logger.lout() << "Expected \""
+		      // << Token::toString(Token::Type::leftBrace)
+		      // << ", found \""
+		      // << Token::toString(curr->type)
+		      // << "\" at "
+		      // << curr->row << ", " << curr->column << ".\n";
```

Comment:
Arrgh, indentation hell! :)

Can you change the commented out code to a part using `#if 0` and `#endif` instead of `//` and again indent it accordingly?

Also, can you change the comment